pkg/gotohelm/helmette/sprig_test.go

```diff
@@ -34,3 +39,156 @@ func TestDefault(t *testing.T) {
 	require.Equal(t, map[string]any{"default": true}, helmette.Default(map[string]any{"default": true}, map[string]any{}))
 	require.Equal(t, map[string]any{"default": false}, helmette.Default(map[string]any{"default": true}, map[string]any{"default": false}))
 }
+
+func TestMerge(t *testing.T) {
+	t.Run("matching_key", func(t *testing.T) {
```

Comment:
super nit: How would you feel about re-writing this as a table driven test?

```go
testCases := []struct{
Name string
Inputs []map[string]any
Output map[string]any
}{
// Cases go here. Notes about what's being exercised can be added as comments here.
}

// Write your test code once
for _, tc := range testCases {
t.Run(tc.Name, func() { ... })
}
```

Doing so makes it a lot easier to add in regression tests and it makes it more clear that you're testing a variety of inputs against the same test code.

Reply:
The test case borrowed from sprig will be harder to model in the table driven test fashion, but I will do my best.

Reply:
Oh gotcha! You're welcome to leave it as is. Nits are always optional.

In theory, it might be fine to just import sprig and re-export their functions in helmette 🤔

Reply:
I feel like test case borrowed from can not be easily added to the table driven tests. I will leave it as stand alone test case.